desc/compute/_omnigenity.py

```diff
@@ -634,6 +634,54 @@ def _interp(x, B):
     )
     B = jnp.moveaxis(B, 0, 1)
     data["|B|"] = B.flatten(order="F")
+    import pdb
+
+    pdb.set_trace()
+    return data
+
+
+@register_compute_fun(
+    name="|B|_pwO",
+    label="|\\mathbf{B}|",
+    units="T",
+    units_long="Tesla",
+    description="Magnitude of omnigenous magnetic field",
+    dim=1,
+    params=["B_min", "B_max", "zeta_C", "theta_C", "t_1", "t_2", "w_2", "iota0", "NFP"],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=[],  # Potential error, we want eq |B|
+    parameterization="desc.magnetic_fields._core.PiecewiseOmnigenousField",
+    parametrization=[],
+)
+def _B_piecewise_omni(params, transforms, profiles, data, **kwargs):
```

Comment:
have it take thetaB etc as data, add a compute param for it for this field, and pass it in with data in the objective using the eq-computed thetab etc

Reply:
I don't think that's possible because then data would have to be a part of the PiecewiseOmnigenity class.